src/pages/TeachersUnite/SaveTheWorldPage.js

```diff
@@ -51,13 +46,11 @@ function SaveTheWorldPage(props) {
                 onPress={() => Navigation.navigate(ROUTES.I_KNOW_A_TEACHER)}
             />
 
-            {!isTeacherAlreadyInvited && (
-                <MenuItem
-                    shouldShowRightIcon
-                    title={translate('teachersUnitePage.iAmATeacher')}
-                    onPress={() => Navigation.navigate(ROUTES.I_AM_A_TEACHER)}
-                />
-            )}
+            <MenuItem
```

Comment:
I removed this condition to allow the teacher to enter again the principal email in case they entered the email incorrectly